src/inmanta/deploy/scheduler.py

```diff
@@ -119,22 +121,42 @@ async def repair(self) -> None:
         pass
 
     async def dryrun(self, dry_run_id: uuid.UUID, version: int) -> None:
-        # FIXME
-        pass
+        resources = await self.build_resource_mappings_from_db(version)
+        for rid, resource in resources.items():
+            self._work.agent_queues.queue_put_nowait(
+                PrioritizedTask(
+                    task=DryRun(
+                        resource=rid,
+                        version=version,
+                        resource_full=resource,
+                        dryrun_id=dry_run_id,
+                    ),
+                    priority=10,
+                )
+            )
 
     async def get_facts(self, resource: dict[str, Any]) -> None:
```

Comment:
```suggestion
    async def get_facts(self, resource: dict[str, object]) -> None:
```

Not from this PR, but I only just noticed it.